infrastructure/adminservices-test/k6tests-rg/providers.tf

```diff
@@ -10,14 +10,11 @@ terraform {
     }
   }
 
-  backend "azurerm" {
-    use_azuread_auth = true
```

Comment:
We should probably do everything in the init action instead of making partial configs.
https://developer.hashicorp.com/terraform/language/backend/azurerm#azure-active-directory-with-openid-connect-workload-identity-federation

Reply:
Opened another PR here: https://github.com/Altinn/altinn-platform/pull/1543